stories/components/teamsChannelPicker/teamsChannelPicker.stories.js

```diff
@@ -84,6 +84,38 @@ export const selectChannel = () => html`
   </script>
 `;
 
+export const clearSelectedItem = () => html`
+  <mgt-teams-channel-picker></mgt-teams-channel-picker>
+
+  <button class="get">Get SelectedChannel</button>
+  <button class="clear">Clear SelectedChannel</button>
+
+  <div class="output"></div>
+
+  <script>
+    document.querySelector('.get').addEventListener('click', _ => {
+      const picker = document.querySelector('mgt-teams-channel-picker');
+      const output = document.querySelector('.output');
+
+      if (picker.selectedItem) {
+        output.innerHTML = '<b>channel:</b> ' + picker.selectedItem.channel.displayName;
+        output.innerHTML += '<br/><b>team:</b> ' + picker.selectedItem.team.displayName;
+      } else {
+        output.innerText = 'no channel selected';
+      }
+    });
+
+    document.querySelector('.clear').addEventListener('click', _ => {
+      const picker = document.querySelector('mgt-teams-channel-picker');
+      const output = document.querySelector('.output');
+      if (picker.selectedItem) {
+        picker.clearSelectedItem();
+        output.innerText = 'no channel selected';
```

Comment:
if we use a selectionChanged event listener then we don't need to set this text here, it will auto update with after the selectionChanged event is fired